pom.xml

```diff
@@ -3988,13 +3988,13 @@
         <jdk>[9,)</jdk>
       </activation>
       <properties>
+        <maven.compiler.release>8</maven.compiler.release>
         <junit.args>
           --add-opens=java.base/java.lang=ALL-UNNAMED
           --add-opens java.base/java.net=ALL-UNNAMED
           --add-opens java.base/java.nio=ALL-UNNAMED
           --add-opens=java.base/java.util=ALL-UNNAMED
           --add-opens java.base/sun.nio.ch=ALL-UNNAMED
-          --add-opens java.security.jgss/sun.security.krb5=ALL-UNNAMED
```

Comment:
I think this line is required. There are other places where this package is used, and build with setting maven.compiler.release to 17 will fail without it.

Reply:
It should work with newer versions for maven.compiler.release since at some point we will have to drop JDK 8 support

Reply:
Thanks I'll check this.

Reply:
@vvysotskyi the module opening lines here are only arguments passed to JUnit, so do you think that if all the unit tests are happy when running in JDK 17 then we are covered here?

Reply:
It is added not only for tests: https://github.com/apache/drill/blob/master/distribution/src/main/resources/drill-config.sh#L317
As I mentioned above, Drill should also work fine with changed maven.compiler.release property to 17 version.